<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanvasCreator</class>
 <widget class="QWidget" name="CanvasCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="widthLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="heightLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>200</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="widthButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set width</string>
   </property>
  </widget>
  <widget class="QPushButton" name="heightButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set height</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
